src/Simplic.Configuration.Data.DB/ConnectionConfigurationRepository.cs

```diff
@@ -0,0 +1,121 @@
+        /// </summary>
+        /// <param name="sqlService"></param>
+        /// <param name="sqlColumnService"></param>
+        public ConnectionConfigurationRepository(ISqlService sqlService, ISqlColumnService sqlColumnService)
+        {
+            this.sqlService = sqlService;
+            this.sqlColumnService = sqlColumnService;
+        }
+
+        /// <summary>
+        /// Deletes a ConnectionConfiguration
+        /// </summary>
+        /// <param name="obj">The object to delete</param>
+        /// <returns>Wherther one connection configuration was deleted</returns>
+        public bool Delete(ConnectionConfiguration obj)
+        {
+            return sqlService.OpenConnection((connection) =>
+            {
+                return connection.Execute($"Delete from {TableName} where ID = :Id", new { obj.Id }) == 1;
+            });
+        }
+
+        /// <summary>
+        /// Deletes a ConnectionConfiguration
+        /// </summary>
+        /// <param name="id">The id of the object to delete</param>
+        /// <returns>Wherther one connection configuration was deleted</returns>
+        public bool Delete(int id)
+        {
+            return sqlService.OpenConnection((connection) =>
+            {
+                return connection.Execute($"Delete from {TableName} where ID = :id", new { id }) == 1;
+            });
+        }
+
+        /// <summary>
+        /// Gets an connection configuration
+        /// </summary>
+        /// <param name="id">the id of the connection configuration</param>
+        /// <returns>The connection configuration instance</returns>
+        public ConnectionConfiguration Get(int id)
+        {
+            return sqlService.OpenConnection((connection) =>
+            {
+                return connection.Query<ConnectionConfiguration>($"Select *, ID as Id, mnd_name as TenantName, mnd_nummer as TenantNumber, dbtype as ConnectionType from {TableName} where ID = :id", new { id }).FirstOrDefault();
+            });
+        }
+
+        /// <summary>
+        /// Gets all connection configurations
+        /// </summary>
+        /// <returns>An enumerable of connection configuration instances</returns>
+        public IEnumerable<ConnectionConfiguration> GetAll()
+        {
+            return sqlService.OpenConnection((connection) =>
+            {
+                return connection.Query<ConnectionConfiguration>($"Select *, ID as Id, mnd_name as TenantName, mnd_nummer as TenantNumber, dbtype as ConnectionType from {TableName}");
+            });
+        }
+
+        /// <summary>
+        /// Saves a connection configuration
+        /// </summary>
+        /// <param name="obj">The object to save</param>
+        /// <returns>Wherther the object saved succesfully</returns>
+        public bool Save(ConnectionConfiguration obj)
+        {
+            if (obj == null)
+                throw new ArgumentNullException(nameof(ConnectionConfiguration));
+
+            try
```

Comment:
I would recomment to check whether obj.Id == 0. If this is true, generate a new id using SELECT Get_Idendity()...